dev-support/ci/selective_ci_checks.sh

```diff
@@ -242,18 +242,25 @@ function get_count_doc_files() {
     start_end::group_end
 }
 
-function get_count_junit_files() {
-    start_end::group_start "Count junit test files"
+function get_count_integration_files() {
+    start_end::group_start "Count integration test files"
     local pattern_array=(
         "^hadoop-ozone/dev-support/checks/_mvn_unit_report.sh"
         "^hadoop-ozone/dev-support/checks/integration.sh"
-        "^hadoop-ozone/dev-support/checks/unit.sh"
+        "^hadoop-ozone/integration-test"
+        "^hadoop-ozone/fault-injection-test/mini-chaos-tests"
         "src/test/java"
-        "src/test/resources"
+    )
+    local ignore_array=(
```

Comment:
I recall you mentioned here:
https://github.com/apache/ozone/pull/2948#issuecomment-1022424382
some helper classes may need to be renamed?

Reply:
I have renamed the ones I found in HDDS-6227 (11605e171).